OpenSim/Simulation/Manager/Manager.cpp

```diff
@@ -870,6 +882,18 @@ void Manager::initialize(SimTK::State& s, double dt )
 }
 //_____________________________________________________________________________
 /**
+* set and initialize a SimTK::TimeStepper
+*/
+void Manager::initializeTimeStepper(
+    const SimTK::System& sys, const SimTK::State& state)
```

Comment:
I agree that the inconsistency is more general, but for the first time I think we have a handle of why that inconsistency was baked in (to handle integrating a CMCActuatorSystem). Correcting it has an impact on the interface being put in place here. I am happy for that to be a follow-up PR, but it could be done here especially `initializeTimeStepper` and the error handling that are being added here and are awkwardly working around the inconsistency. @carmichaelong it is up to you. As long as the we fix the inconsistency soon, either way is fine with me.